CRM/Upgrade/Incremental/sql/5.56.alpha1.mysql.tpl

```diff
@@ -1,5 +1,23 @@
 {* file to handle db changes in 5.56.alpha1 during upgrade *}
 
+-- Add in Year prior to previous fiscal year and Previous 2 fiscal years
+SELECT @option_group_id_date_filter := max(id) from civicrm_option_group where name = 'relative_date_filters';
+UPDATE `civicrm_option_value` SET weight = 64 WHERE option_group_id = @option_group_id_date_filter AND weight = 62;
+UPDATE `civicrm_option_value` SET weight = 63 WHERE option_group_id = @option_group_id_date_filter AND weight = 61;
+UPDATE `civicrm_option_value` SET weight = 62 WHERE option_group_id = @option_group_id_date_filter AND weight = 60;
+UPDATE `civicrm_option_value` SET weight = 61 WHERE option_group_id = @option_group_id_date_filter AND weight = 59;
+UPDATE `civicrm_option_value` SET weight = 59 WHERE option_group_id = @option_group_id_date_filter AND weight = 58;
+UPDATE `civicrm_option_value` SET weight = 58 WHERE option_group_id = @option_group_id_date_filter AND weight = 57;
+UPDATE `civicrm_option_value` SET weight = 57 WHERE option_group_id = @option_group_id_date_filter AND weight = 56;
+UPDATE `civicrm_option_value` SET weight = 56 WHERE option_group_id = @option_group_id_date_filter AND weight = 55;
+UPDATE `civicrm_option_value` SET weight = 55 WHERE option_group_id = @option_group_id_date_filter AND weight = 54;
+INSERT INTO
+   `civicrm_option_value` (`option_group_id`, `label`, `value`, `name`, `grouping`, `filter`, `is_default`, `weight`, `description`, `is_optgroup`, `is_reserved`, `is_active`, `component_id`, `visibility_id`, `icon`)
+VALUES
+   (@option_group_id_date_filter, {localize}'{ts escape="sql"}Previous 2 fiscal years{/ts}'{/localize}, 'previous_2.fiscal_year', 'previous_2.fiscal_year', NULL, NULL, 0,54, NULL, 0, 0, 1, NULL, NULL, NULL),
```

Comment:
I'm not really familiar with `{localize}`, but it looks like you'd need to also localize the INSERT list, e.g.

`INSERT INTO INTO civicrm_option_value (option_group_id, {localize field='label'}label{/localize}`

Reply:
Done.